HelloWorld-CMake/Source/HelloWorld/HelloWorld.swift

```diff
@@ -14,7 +14,7 @@ func display(_ message: String) -> Bool {
 
 func main() {
   let accessor: HelloWorldStringAccessor = HelloWorldStringAccessor()
-  guard let message = try? accessor.access(), display(message!) else {
```

Comment:
How did this build previously? Thanks for fixing this.  If you want to split this into a separate change, we can merge this right away.